spec/support/disbursement_system_helpers.rb

```diff
@@ -0,0 +1,11 @@
+module DisbursementSystemHelpers
+  def verify_disbursement_item
+    within('.govuk-table__row', text: 'Apples') do
```

Comment:
not sure having this hear is helping, any reason for not leaving this inline in the test?

Reply:
Removed this helper for now, we will revisit as part of cleanup.